Userland/Applications/Browser/Tab.cpp

```diff
@@ -57,12 +57,14 @@ URL url_from_user_input(const String& input)
 
 void Tab::start_download(const URL& url)
```

Comment:
If you touch files quite extensively, it would be nice if you would convert them to east const style (in a separate commit). :^)
```suggestion
void Tab::start_download(URL const& url)
```